### PR TITLE
CMLK-1314 - Upgrade x/net to version 0.17.0

### PR DESCRIPTION
**Pull request by oscerd**
**Release Note**
```release-note
CMLK-1314 - Upgrade x/net to version 0.17.0
```

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -611,8 +610,8 @@ golang.org/x/text/unicode/norm
 # golang.org/x/time v0.0.0-20220224211638-0e9765cccd65
 ## explicit
 golang.org/x/time/rate
-# golang.org/x/tools v0.1.10-0.20220218145154-897bd77cd717
-## explicit; go 1.17
+# golang.org/x/tools v0.6.0
+## explicit; go 1.18
```
Comment (gansheer) [inline vendor/modules.txt:614]: Should this reference to 1.18 be there ?

Comment (oscerd) [reply vendor/modules.txt:614]: Yes.

**Review by gansheer — APPROVED** [review]

**Comment by oscerd** [comment]
Tested with golang 1.17.13.

**Comment by squakez** [comment]
LGTM. For the records, it is not required to update the vendor. We have an automatic process taking care of that.

**Comment by oscerd** [comment]
Compiling will fail if don't go mod vendor unless we use mod read-only I guess

**Comment by squakez** [comment]
> Compiling will fail if don't go mod vendor unless we use mod read-only I guess

True. Local build still requires it.

**Pull request merged**